public/docs/getting_started/compile.md

```diff
@@ -1,43 +1,59 @@
-# Location
+# Compiling contracts
 
-All of your contracts are located in the `./contracts` directory. By default you're given an example Solidity [contract](http://solidity.readthedocs.org/en/latest/contracts.html) file and an example Solidity [library](http://solidity.readthedocs.org/en/latest/contracts.html#libraries) file, both with filenames ending in `.sol`. Although a Solidity library is different than a contract, for documentation purposes we're going to lump these into the same word, "contracts".
+## Location
 
-# Command
+All of your contracts are located in your project's `contracts/` directory. As contracts are written in [Solidity](LINK), all files containing contracts will have a file extension of `.sol`. Associated Solidity [libraries](http://solidity.readthedocs.org/en/latest/contracts.html#libraries) will also have a `.sol` extension.
```

Comment:
Whoops. Looks like there's a LINK here that's not fleshed out. Probably this link is best? https://solidity.readthedocs.io/en/develop/